src/ImageSharp.Processing/Transforms/ResizeProcessor.cs

```diff
@@ -3,7 +3,7 @@
 // Licensed under the Apache License, Version 2.0.
 // </copyright>
 
-namespace ImageSharp.Processors
```

Comment:
Can we put all these processors back please. I don't understand why they were all moved.